packages/plugins/typescript/operations/src/config.ts

```diff
@@ -297,8 +297,14 @@ export interface TypeScriptDocumentsPluginConfig extends RawDocumentsConfig {
    * @description Options related to handling nullability
    * @exampleMarkdown
    * ## `errorHandlingClient`
+   * An error handling client is a client which prevents the user from reading a `null` used as a placeholder for an error in a GraphQL response.
+   * The client may do so by throwing when an errored field is accessed (as is the case for `graphql-toe`),
```

Comment:
Should we link to `graphql-toe` doc here to help our readers find out what it does?

Reply:
```suggestion
   * The client may do so by throwing when an errored field is accessed (as is the case for [`graphql-toe`](https://github.com/graphile/graphql-toe)),
```